packages/core/admin/admin/src/components/GlobalStyle/index.js

```diff
@@ -1,11 +1,17 @@
-import 'sanitize.css/sanitize.css';
-import 'bootstrap/dist/css/bootstrap.css';
-import 'font-awesome/css/font-awesome.min.css';
-import '@fortawesome/fontawesome-free/css/all.css';
-// eslint-disable-next-line import/extensions
-import '@fortawesome/fontawesome-free/js/all.min.js';
 import { createGlobalStyle } from 'styled-components';
 
+const loadCss = async () => {
+  await import(/* webpackChunkName: "sanitize-css" */ 'sanitize.css/sanitize.css');
+  await import(/* webpackChunkName: "fontawesome-css" */ 'font-awesome/css/font-awesome.min.css');
+  await import(
+    /* webpackChunkName: "fontawesome-css-all" */ '@fortawesome/fontawesome-free/css/all.css'
+  );
+  await import(/* webpackChunkName: "fontawesome-js" */ '@fortawesome/fontawesome-free/js/all.min');
+  await import(/* webpackChunkName: "bootstrap" */ 'bootstrap/dist/css/bootstrap.css');
```

Comment:
WYT of not loading these async? The DS will solve the problem quite soon no?

Reply:
Yep we will remove these import during the DS setup